python_ta/checkers/function_parameter_not_mentioned_checker.py

```diff
@@ -0,0 +1,59 @@
+"""checker that every function parameter is mentioned by name in the docstring text.
+"""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class FunctionParameterNotMentionedChecker(BaseChecker):
+    """
+    A class checker to check if every function parameter is mentioned by name within it's the docstring.
```

Comment:
"A class checker" -> "A class"

"within it's the docstring" -> "within the function's docstring"